setup.ts

```diff
@@ -0,0 +1,196 @@
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
1. Please remove. Fix the cause of the linting error.
1. We need our linting rules to be followed. We have created an issue to enforce this as part of future PR workflows

Reply:
Removed